### ISSUE TITLE
Dataref could not be parsed

### ISSUE THREAD

**Issue by DevolperRL** [X-Plane/XPlane2Blender#725]
Hi,
after I installed this version(4.2) and blender 3.2, I got this message:
![image](https://user-images.githubusercontent.com/100154298/222502269-40f7e7a5-a321-481e-b9ce-b7a3caed232a.png)
Can someone help me please how to fix this? I would continue with my plane

**Comment by luweejinator** [comment]
same.

**Comment by luweejinator** [comment]
i fixed it by removing the xplane2blender version 4.2-3 from edit>preferences>addons in blender and then installing the io_xplane2blender_4_2_0-rc_2-117_20220419015723-v12.zip
i can "parse datarefs" now...

**Comment by luweejinator** [comment]
i tried 4.2 rc 3 again..  this time i didnt uninstall 4.2 rc 2 and clicked install and selected the rc3 zip and it didnt break anything...

**Comment by luweejinator** [comment]
not sure what happened this time... 

Python: Traceback (most recent call last):
  File "C:\Users\luijee\AppData\Roaming\Blender Foundation\Blender\3.2\scripts\addons\io_xplane2blender\xplane_ops.py", line 882, in execute
    bake_temp_folder.mkdir(parents=True, exist_ok=True)
  File "C:\Program Files\Blender Foundation\Blender 3.2\3.2\python\lib\pathlib.py", line 1173, in mkdir
    self._accessor.mkdir(self, mode)
PermissionError: [WinError 5] Access is denied: '_tmp_bake_images'

**Comment by luweejinator** [comment]
closed blender...
opened as administrator...  error went away...

**Comment by ygtshn** [comment]
Hello guys, I am having the same exact problem on linux. Tried running blender as root but did not work. I am on blender 3.3.5 and xplane2blender 4.2.0. Tried multiple deleting/reinstalling actions but did not succeed. Could you help?

**Comment by hltdev8642** [comment]
So I was able to resolve this by modifying the DataRefs.txt file. 

Seems (at least for me) that the error is occuring due to lines that do not have exactly 4 tab-seperated values. 

The way I fixed it, was by:
1) adding: `[tab]???[tab]???` to any lines with **less than 3 tabs**.
2) removing extra tabs from lines with **more than 3 tabs**.  (note: this mainly occurs on lines with DEPRECIATED or other items at the end of lines in all CAPS.)
3) to start, I replaced my DataRefs.txt file with the one located in the X-Plane folder. On doing this for some reason through process of elimination figured out that the following line single-handedly caused parsing to fail: `sim/operation/misc/commandline	byte[]	n	strings	A consecutively packed series of null terminated C strings - read array size to find total length of all of them.  Contains each separated command line argument (So you may want to find/replace and remove this line as well`

I will attach my DataRefs.txt file so people are welcome to just use that and remove the need to troubleshoot, but for those who want to fix their own DataRefs.txt file I used the following command that counts the number of tabs per line:
```
awk '{print gsub(/\t/,"")}' DataRefs.txt  > output.txt
``` 
Basically you can run this and it outputs the number of tabs on each line to a text file, allowing you to see which lines are missing / have too many tabs on them. I used grep after personally: `grep '4' --invert-match output.txt`  (this will result in all the lines that have more/less than the required number of tabs (4). 

Hope this helps, works for me on the latest xplane2blender! I also don't mind helping if anyone is stuck/confused.

- hltdev

**Comment by hltdev8642** [comment]
Here is the dataref file:

[DataRefs.txt](https://github.com/X-Plane/XPlane2Blender/files/11705809/DataRefs.txt)

**Comment by bsupnik** [comment]
@hltdev8642 this looks like a bug in the exporter's parser - we will investigate!

**Comment by merlins8** [comment]
> Here is the dataref file:
> 
> [DataRefs.txt](https://github.com/X-Plane/XPlane2Blender/files/11705809/DataRefs.txt)
> 
> For anyone who does not know where to put this for some reason, it replaces the `DataRefs.txt` file located at:
> 
> ```
> %APPDATA%\Blender Foundation\Blender\[BLENDER-VERSION]\scripts\addons\io_xplane2blender\resources\
> ```

didnt work

**Comment by Arkan-Seng** [comment]
> > Here is the dataref file:
> > [DataRefs.txt](https://github.com/X-Plane/XPlane2Blender/files/11705809/DataRefs.txt)
> > For anyone who does not know where to put this for some reason, it replaces the `DataRefs.txt` file located at:
> > ```
> > %APPDATA%\Blender Foundation\Blender\[BLENDER-VERSION]\scripts\addons\io_xplane2blender\resources\
> > ```
> 
> didnt work

Hello, it works perfectly, I've just tested it on Blender 4.0. I don't think you've placed the file in the right place. In my case it is here:

C:\Users\[YOURACCOUNTNAME]\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\io_xplane2blender\resources
